app/web/features/search/SearchBox.tsx

```diff
@@ -118,25 +119,29 @@ export default function SearchBox({
     setValue("query", searchFilters.active.query ?? "");
   }, [setValue, searchFilters.active.location, searchFilters.active.query]);
 
-  const filterDialogButton = (
-    <>
-      <Button
-        onClick={() => setIsFiltersOpen(true)}
-        className={classNames(className, classes.filterDialogButton)}
-        variant={isSmDown ? "contained" : "outlined"}
-        size="small"
-      >
-        {FILTER_DIALOG_TITLE}
-      </Button>
-      <FilterDialog
-        isOpen={isFiltersOpen}
-        onClose={() => setIsFiltersOpen(false)}
-        searchFilters={searchFilters}
-      />
-    </>
+  const filterDialog = (
+    <FilterDialog
+      isOpen={isFiltersOpen}
+      onClose={() => setIsFiltersOpen(false)}
+      searchFilters={searchFilters}
+    />
   );
+
   if (isSmDown) {
-    return filterDialogButton;
+    return (
+      <>
+        <Button
+          onClick={() => setIsFiltersOpen(true)}
+          className={classNames(className)}
+          variant="contained"
+          size="small"
+        >
+          {FILTER_DIALOG_TITLE_MOBILE}
+        </Button>
```

Comment:
Do we want the button to stretch almost the full width of the map, rather than sticking next to the +/- buttons as before?

Reply:
Hmm. You mean like moving the +/- buttons a bit below, so the "Search" button has full width? I hadn't heard this idea before

Reply:
Sorry for the slow response! I meant literally as the styling before where the button is not stretched and placed to the left of the +/- buttons.

To rephrase, the question perhaps should have been "Is stretching the button (your change) a desirable behaviour?"

Reply:
Ah I see. I didn't change that, I think 🤔 The button is already stretched in the live site:

![Screen Shot 2022-05-15 at 15 26 22](https://user-images.githubusercontent.com/1557348/168475207-9e482f04-5bea-4301-81f5-5bc1db149d19.png)

Is there a difference, apart from the button label, that I'm not seeing?

Reply:
It used to not be stretched! 😆 
I think not stretched looks a little nicer

Reply:
I see! No problem, I can make it not stretched then, and placed on the left of the +/- buttons. Will do in this same branch soon

Reply:
Here's what it looks like aligned next to the zoom buttons without stretching:

![Screen Shot 2022-05-16 at 09 28 12](https://user-images.githubusercontent.com/1557348/168540995-f12d77db-22fb-46ba-a0fe-8d12aff50bbe.png)

Let me know!